.github/workflows/nightly-scans.yaml

```diff
@@ -193,20 +229,58 @@ jobs:
           ]
     steps:
       - uses: actions/checkout@v3
+      - name: Login with ghcr
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.repository_owner }}
+          password: ${{ secrets.GITHUB_TOKEN }}
       - name: Install yq
         run: |
           sudo snap install yq
       - uses: ./.github/actions/k8s-version-config
         name: Setup k8s cluster
         with:
           k8s-version: ${{ matrix.k8s-version }}
-          load-local-image: false
       - name: Run pre-config and workload integration tests
         run: |
-          bash tests/integration/integration-test.sh "nightly-pre-and-workload"
```

Comment:
is the `"nightly-pre-and-workload"` step in the integration test still needed after those changes?

Reply:
why would it not? This will test the latest build

Reply:
oh, hold on this may have been a mishap